docs/BreakingChanges/119 ADO.NET now attempts to automatically reconnect broken SQL connections.md

```diff
@@ -19,11 +19,11 @@ Beginning in the .NET Framework 4.5.1, the .NET Framework will attempt to automa
 If this feature is undesirable due to compatibility concerns, it can be disabled by setting the ConnectRetryCount property of a connection string (or SqlConnectionStringBuilder) to 0.
 
 ### Affected APIs
-* `M:System.Data.IDbConnection.set_ConnectionString(System.String)`
-* `M:System.Data.SqlClient.SqlConnection.set_ConnectionString(System.String)`
-* `M:System.Configuration.ConnectionStringSettings.set_ConnectionString(System.String)`
-* `M:System.Data.Common.DbConnection.set_ConnectionString(System.String)`
-* `M:System.Data.Common.DbConnectionStringBuilder.set_ConnectionString(System.String)`
+* `P:System.Data.IDbConnection.ConnectionString`
+* `P:System.Data.SqlClient.SqlConnection.ConnectionString`
+* `P:System.Configuration.ConnectionStringSettings.ConnectionString`
+* `P:System.Data.Common.DbConnection.ConnectionString`
+* `P:System.Data.Common.DbConnectionStringBuilder.ConnectionString`
```

Comment:
Similar to my last comment, these breaks are specifically in how setting the connection string is handled, so it may be better to only list the set methods.